content/plugins/provide-plugin.md

```diff
@@ -38,3 +41,11 @@ new webpack.ProvidePlugin({
   'window.jQuery': 'jquery'
 })
 ```
+
+### Use Vue.js(ES Modules)
+
+```javascript
+new webpack.ProvidePlugin({
+  Vue: ['vue/dist/vue.esm.js', 'default'] // v2.2.6
```

Comment:
Confirmed that this is an unexpected behaviour of webpack, see https://github.com/webpack/webpack/issues/4729. Since this is independent of Vue or Babel, I'd suggest to have a [warning box](https://webpack.js.org/writers-guide/#tip) just above the "Typical use-cases" explaining the current need to specify the `"default"` for ES2015 modules.

If you want to keep the Vue section, please remove the "(ES Modules)" from the caption and the version comment.